src/main/java/org/sonar/plugins/stash/StashPlugin.java

```diff
@@ -38,38 +42,68 @@
   public static final String STASH_ISSUE_THRESHOLD = "sonar.stash.issue.threshold";
   public static final String STASH_TIMEOUT = "sonar.stash.timeout";
   public static final String SONARQUBE_URL = "sonar.host.url";
-  
+  public static final String STASH_INCLUDE_ANALYSIS_OVERVIEW = "sonar.stash.include.analysis.overview";
+  public static final String STASH_INCLUDE_EXISTING_ISSUES = "sonar.stash.include.existing.issues";
+  public static final String STASH_INCLUDE_VICINITY_ISSUES = "sonar.stash.include.vicinity.issues";
+
   @Override
   public List getExtensions() {
     return Arrays.asList(
-        StashIssueReportingPostJob.class,
-        StashPluginConfiguration.class,
-        InputFileCache.class,
-        InputFileCacheSensor.class,
-        StashProjectBuilder.class,
-        StashRequestFacade.class,
-        PropertyDefinition.builder(STASH_URL)
-            .name("Stash base URL")
-            .description("HTTP URL of Stash instance, such as http://yourhost.yourdomain/stash")
-            .subCategory(CONFIG_PAGE_SUB_CATEGORY_GENERAL)
-            .onQualifiers(Qualifiers.PROJECT).build(),
-        PropertyDefinition.builder(STASH_LOGIN)
-            .name("Stash base User")
-            .description("User to push data on Stash instance")
-            .subCategory(CONFIG_PAGE_SUB_CATEGORY_GENERAL)
-            .onQualifiers(Qualifiers.PROJECT).build(),
-        PropertyDefinition.builder(STASH_TIMEOUT)
-            .name("Stash issue Timeout")
-            .description("Timeout when pushing a new issue to Stash (in ms)")
-            .subCategory(CONFIG_PAGE_SUB_CATEGORY_GENERAL)
-            .onQualifiers(Qualifiers.PROJECT)
-            .defaultValue(DEFAULT_STASH_TIMEOUT_VALUE).build(),
-        PropertyDefinition.builder(STASH_ISSUE_THRESHOLD)
-            .name("Stash issue Threshold")
-            .description("Threshold to limit the number of issues pushed to Stash server")
-            .subCategory(CONFIG_PAGE_SUB_CATEGORY_GENERAL)
-            .onQualifiers(Qualifiers.PROJECT)
-            .defaultValue(DEFAULT_STASH_THRESHOLD_VALUE).build());
+            StashIssueReportingPostJob.class,
+            StashPluginConfiguration.class,
+            InputFileCache.class,
+            InputFileCacheSensor.class,
+            StashProjectBuilder.class,
+            StashRequestFacade.class,
+            PropertyDefinition.builder(STASH_URL)
+                .name("Stash URL")
+                .description("HTTP URL of Stash, such as http://yourhost.yourdomain/stash")

```

Comment:
We might remove the specification of protocol as it can be HTTPS as well =>
URL of Stash, such as http:....